string[] diagnostics
float32 speed_setpoint
float32 current_speed
float32 voltage
float32 motor_current_draw
float32 distance

uint8 OK=0
uint8 WARN=1
uint8 CRITICAL=2

uint8 distance_state
uint8 motor_current_state
uint8 battery_state